CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing to the Repository
+
+We've put together the following guidelines to help you figure out where you can best be helpful.
+
+## Table of Contents
+
+1. [Desired Contributions](#desired-contributions)
+2. [How to Contribute](#how-to-contribute)
```

Comment:
In markdown you can do 1. for all items in a numbered list, and it will automatically number them. It makes it easier to switch their order around, add new items, etc.